js/ui/speed_dial_action/speed_dial_main_item.js

```diff
@@ -130,15 +140,25 @@ const SpeedDialMainItem = SpeedDialItem.inherit({
                 this._clickHandler();
             });
 
+
```

Comment:
I suggest moving this arithmetical code code to separate method like _calculatePosition